assets/js/applicationinsights.js

```diff
@@ -0,0 +1,22 @@
+window.applicationInsights = (function () {
```

Comment:
This file adds an object `applicationInsights` to the `window` object (essentially a global variable) with a single method `init`
This syntax is a _bit like_ a class (except of course that classes don't exist in javascript - I think this is called the "revealing module pattern"). Anyway, think of it a bit like a class, which currently has one method `init`
I've written it this way because I'm fully expecting us to have to extend and build upon this in the future by adding additional methods .... such as `trackEvent`

In its current form, with the call to `trackPageView` on line 18, it will record an app insights `pageView` event for every page render.